sphinx/source/conf.py

```diff
@@ -66,6 +70,12 @@
 
 bokeh_missing_google_api_key_ok = False
 
+# Check for GOOGLE_API_KEY
```

Comment:
This comment is redundant / unnecessary. 

I'm ambivalent but another idea would be to move the code below to a function in `bokeh.settings` (just to make this conf file a little shorter/cleaner) e.g `check_google_api_key_env()`

Reply:
@bryevdv I'm not sure about the file you're referring to, is it: https://github.com/bokeh/bokeh/blob/branch-2.3/bokeh/settings.py?